lib/utils/logger.js

```diff
@@ -15,7 +17,8 @@ class Logger {
 
     this.config = Object.assign({
       context: 'sequelize',
-      debug: true
+      debug: true,
+      inspector: value => util.inspect(value, false, 3)
```

Comment:
No need to pass this in config I think, just use this as default in the new `inspect` method

Reply:
Agree